addons/pos_loyalty/static/src/tours/PosLoyaltyTourMethods.js

```diff
@@ -142,7 +142,7 @@ odoo.define('pos_loyalty.tour.PosCouponTourMethods', function (require) {
                 {
                     trigger: isHighlighted
                         ? '.control-button.highlight:contains("Reward")'
-                        : '.control-button:contains("Reward"):not(:has(.highlight))',
+                        : '.control-button:contains("Reward"):not(.highlight)',
```

Comment:
This was actually not working when checking if the button was not highlighted

Reply:
Interesting. I'm always confused whether we need ":has" or not. Anyway, if this change will make the assertion work, I'm okay with this change.